```
aasdk/src/Transport/SSLWrapper.cpp:55:43: error: implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
    auto bio = BIO_new_mem_buf(certificate.c_str(), certificate.size());
```

--- a/src/Transport/SSLWrapper.cpp
+++ b/src/Transport/SSLWrapper.cpp
@@ -16,6 +16,7 @@
 *  along with aasdk. If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <string>
 #include <openssl/engine.h>
 #include <openssl/err.h>
 #include <openssl/ssl.h>

